Affine Cipher.py

```diff
@@ -1,38 +1,53 @@
 # v0.9
+# infinite loop
+while 0 == 0:
+    result = ""
```

Comment:
what is this for?

Reply:
It makes the program run again after the encoding is finished. It just ends up being a nicer experience.

Reply:
Doesn't is constantly set `result` to ` `? So how can you set `result` to anything else to give output?